frontend/src/types/generalTypes.ts

```diff
@@ -0,0 +1,8 @@
+export type CorrelationFilter = {
+  pLow: boolean;
+  pMedium: boolean;
+  pHigh: boolean;
+  nLow: boolean;
+  nMedium: boolean;
+  nHigh: boolean;
+};
```

Comment:
Does this need it's own file? Can't it be placed under react/componentTypes.ts?

Reply:
hmm, Andreas meant that we should create a new file.

Reply:
Ok then I guess it's fair 👍